airflow/providers/airbyte/hooks/airbyte.py

```diff
@@ -257,3 +257,39 @@ def test_connection(self):
             return False, str(e)
         finally:
             self.method = "POST"
+
+    def get_airbyte_connection_info(self, connection_id: str):
```

Comment:
I'd love to see some type hints here for the return type and maybe some more information about what we are actually getting from it? I think it should return structured connection information and not a generic response. All the error handling etc. should be handled within the method, just like in the above `get_job_details`. As it's in the hook, it should be useful for all kinds of further use, not only OpenLineage.

Reply:
I think this one is resolved.